lib/Runtime/ByteCode/ByteCodeSerializer.cpp

```diff
@@ -64,7 +64,7 @@ namespace Js
     const int magicStartOfPropIdsOfFormals = *(int*)"pif[";
     const int magicEndOfPropIdsOfFormals = *(int*)"]pif";
     const int magicStartOfSlotIdToNestedIndexArray = *(int*)"sni[";
-    const int magicEndOfSlotIdToNestedIndexArray = *(int*)"]sni"
+    const int magicEndOfSlotIdToNestedIndexArray = *(int*)"]sni";
```

Comment:
>; [](start = 64, length = 1)

wait... how was this not an error?

Reply:
The code is only enabled explicitly to debug issues.  No official build has this code enabled.